spec/changelog.yml

```
@@ -15,6 +15,11 @@
     - description: Require defining agent version constraints in input and integration packages.
       type: breaking-change
       link: https://github.com/elastic/package-spec/pull/999
+- version: 3.5.4-next
```

```suggestion
- version: 3.5.3-next
```

are we skipping a patch?